pkg/action/install_test.go

```diff
@@ -361,7 +363,62 @@ func TestInstallRelease_Wait(t *testing.T) {
 	is.Contains(res.Info.Description, "I timed out")
 	is.Equal(res.Info.Status, release.StatusFailed)
 }
+func TestInstallRelease_Wait_Interrupted(t *testing.T) {
+	if os.Getenv("HANDLE_SIGINT") == "1" {
```

Comment:
What if the user had already set HANDLE_SIGINT for some other program's execution?

Reply:
With the usage of context.Context this issue was removed